osv/tests.py

```diff
@@ -239,3 +240,58 @@ def mock_clone(test, func=None, return_value=None):
   else:
     mocked.side_effect = func
   test.addCleanup(patcher.stop)
+
+
+def setup_gitter(env_var):
```

Comment:
can you turn this into a context manager like the [datastore emulator](https://github.com/google/osv.dev/blob/6252d8b238d245c3008c3870d29f1e9ef4f15751/osv/tests.py#L137-L138), and use `unittest.enterModuleContext()` instead of manually closing it in the teardown